moveit_planners/ompl/ompl_interface/src/ompl_planner_manager.cpp

```diff
@@ -79,16 +84,16 @@ class OMPLPlannerManager : public planning_interface::PlannerManager
           case ompl::msg::LOG_DEV2:
           case ompl::msg::LOG_DEV1:
           case ompl::msg::LOG_DEBUG:
-            OMPL_ROS_LOG(::ros::console::levels::Debug);
```

Comment:
@mamoll Do you know why this package uses the different Logging macros?

Reply:
I'm all for standardizing to the RCLCPP ones as they filter better

Reply:
I think the usual ROS logging macros wouldn't allow you to change the logging name (i.e. filename, line) because it's stored in a static variable. I'm not sure if the RCLCPP approach I added prints nice debug messages, but I'll test this when running the MoveItCpp integration

Reply:
@mlautman OMPL is trying to be ROS-agnostic. It has its logging infrastructure, but can be easily hooked into the ROS logging infrastructure. (Not a coincidence, BTW, since Ioan Sucan wrote the original version of logging for both, I believe.)